lib/carthage_cache/configuration.rb

```diff
@@ -32,7 +32,7 @@ def self.default
           secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'],
           profile: ENV['AWS_PROFILE']
         },
-        tmpdir: Dir.tmpdir
+        tmpdir: File.join(Dir.home, 'Library', 'Caches')
```

Comment:
What about having a `carthage_cache` subdirectory inside `~/Library/Caches/`? I think it would be better for debugging and organizing files.